src/script/components/InputBar/InputBar.tsx

```diff
@@ -599,6 +599,7 @@ export const InputBar = ({
               </div>
               {!isSelfUserRemoved && !pastedFile && (
                 <RichTextEditor
+                  conversation={conversation}
```

Comment:
As I see we only use the conversation's id there, let's not pass the whole object, but just the id

```suggestion
                  conversationId={conversation.id}
```